setup.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import setuptools
+
+# Change directory to allow installation from anywhere
+script_folder = os.path.dirname(os.path.realpath(__file__))
+os.chdir(script_folder)
+
+with open("requirements.txt") as f:
+    requirements = f.read().splitlines()
+
+# Installs
+setuptools.setup(
+    name='nuplan-garage',
+    version='1.0.0',
+    author='CS_Tu @ University of Tuebingen',
+    author_email='kashyap.chitta@uni-tuebingen.de',
```

Comment:
@kashyap7x should we keep your email here (same one as in the readme), or should we create a separate inbox for the nuplan-garage?